planner/matchmaker/pom.xml

```diff
@@ -30,8 +30,35 @@ limitations under the License.
     <dependencies>
         <dependency>
             <groupId>eu.seaclouds-project</groupId>
-            <artifactId>toscamodel</artifactId>
-            <version>1.0.0</version>
```

Comment:
I left it there because of optimizer referencing to it. Otherwise CI complains. It should be removed when we have the updated optimized.